addons/mail/tests/discuss/test_ui.py

```diff
@@ -9,7 +9,7 @@
 class TestUi(HttpCaseWithUserDemo):
 
     def test_01_mail_tour(self):
-        self.start_tour("/web", 'discuss_channel_tour', login="admin")
+        self.start_tour("/odoo", 'discuss_channel_tour', login="admin")
```

Comment:
Couldn't we have a webclient root url exported somewhere so that we don't need to hardcode the path everywhere?

Reply:
`stateToUrl` can be also used in this case : 
```js
expect(stateToUrl({})).toBe("/odoo");
```